tests/test_validator.py

```diff
@@ -61,6 +61,30 @@ def _create_full_match(matches):
             + r"$"  # match no more matches
         )
 
+    @pytest.mark.parametrize(
+        "decl, matches",
+        [
+            (
+                "gate cnot[0, 0];",
+                [
+                    "Declaration 'gate cnot[0, 0]' has duplicate wires labels.",
+                ],
+            ),
+        ],
+    )
+    def test_message_reset(self, decl, matches):
+        """Tests that validator messages are reset between different runs."""
```

Comment:
Why did you choose to parametrize this test?
```suggestion
    def test_message_reset(self, decl, matches):
        """Tests that validator messages are reset between different runs."""
        decl = "gate cnot[0, 0];"
        matches = ["Declaration 'gate cnot[0, 0]' has duplicate wires labels."]
```

Reply:
I just copied another test that was parametrized. No other reason.